config/webpack.config.dev.js

```diff
@@ -1,38 +1,44 @@
-var autoprefixer = require('autoprefixer');
-var cssvars = require('postcss-simple-vars');
-var webpack = require('webpack');
-var HtmlWebpackPlugin = require('html-webpack-plugin');
-var CaseSensitivePathsPlugin = require('case-sensitive-paths-webpack-plugin');
-var InterpolateHtmlPlugin = require('react-dev-utils/InterpolateHtmlPlugin');
-var WatchMissingNodeModulesPlugin = require('react-dev-utils/WatchMissingNodeModulesPlugin');
-var getClientEnvironment = require('./env');
-var paths = require('./paths');
+const autoprefixer = require('autoprefixer');
+const cssvars = require('postcss-simple-vars');
+const webpack = require('webpack');
+const HtmlWebPackPlugin = require("html-webpack-plugin");
 
+const paths = require('./paths');
+const getClientEnvironment = require('./env');
 
+const publicPath = '/';
 
-// Webpack uses `publicPath` to determine where the app is being served from.
-// In development, we always serve from the root. This makes config easier.
-var publicPath = '/';
-// `publicUrl` is just like `publicPath`, but we will provide it to our app
+// `publicUrl` we will provide it to our app
 // as %PUBLIC_URL% in `index.html` and `process.env.PUBLIC_URL` in JavaScript.
 // Omit trailing slash as %PUBLIC_PATH%/xyz looks better than %PUBLIC_PATH%xyz.
 var publicUrl = '';
 // Get environment variables to inject into our app.
 var env = getClientEnvironment(publicUrl);
 
-var cssvariables = require(paths.appSrc + '/theme/variables');
+const cssvariables = require(paths.appSrc + '/theme/variables');
+
+const postcssPlugins = [
+  autoprefixer({
+    browsers: [
+      '>1%',
+      'last 4 versions',
+      'Firefox ESR',
+      'not ie < 9', // React doesn't support IE8 anyway
+    ]
+  }),
+  cssvars({
+    variables: function () {
+        return Object.assign({}, cssvariables);
+    },
+    silent: false
+  }),
+];
 
-// This is the development configuration.
-// It is focused on developer experience and fast rebuilds.
-// The production configuration is different and lives in a separate file.
 module.exports = {
-  // You may want 'eval' instead if you prefer to see the compiled output in DevTools.
-  // See the discussion in https://github.com/facebookincubator/create-react-app/issues/343.
   devtool: 'cheap-module-source-map',
```

Comment:
why not `eval-source-map`? https://webpack.js.org/configuration/devtool/

Reply:
Basically because at this point it is enough, but yeah probably ```eval-source-map``` is better. I will update it in my next frontend PR